components/script/dom/xmlhttprequest.rs

```diff
@@ -1051,11 +1051,11 @@ impl XMLHttpRequest {
                 self.upload_complete.set(true);
                 // Substeps 2-4
                 if !self.sync.get() {
-                    self.dispatch_upload_progress_event(atom!("progress"), None);
+                    self.dispatch_upload_progress_event(atom!("progress"), Err(()));
                     return_if_fetch_was_terminated!();
-                    self.dispatch_upload_progress_event(atom!("load"), None);
+                    self.dispatch_upload_progress_event(atom!("load"), Err(()));
                     return_if_fetch_was_terminated!();
-                    self.dispatch_upload_progress_event(atom!("loadend"), None);
+                    self.dispatch_upload_progress_event(atom!("loadend"), Err(()));
```

Comment:
The changes here should not be passing an error value; this is the success case.